projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -1491,17 +1491,63 @@ public function render_date_field( $id, $label, $value, $class, $required, $requ
 		}
 
 		Assets::register_script(
-			'grunion-frontend',
-			'../../dist/contact-form/js/grunion-frontend.js',
+			'jp-date-picker',
```

Comment:
```suggestion
			'jp-forms-date-picker',
```

...to avoid too generic handle.